composer.json

```diff
@@ -17,7 +17,7 @@
         "phpcsstandards/phpcsdevtools": "^1.1.0",
         "phpcompatibility/php-compatibility": "^9.3.5",
         "php-parallel-lint/php-parallel-lint": "^1.3.2",
-        "phpunit/phpunit": "^7.5.20",
+        "phpunit/phpunit": "^4.0 || ^5.0 || ^6.0 || ^7.0 || ^8.0 || ^9.0",
```

Comment:
How come this adds version as low as 4.0?

Reply:
We should never have pinned it to 7.5.* in the first place. Our standard technically doesn't support a PHP version below 7.2 — even though the code should function fine in lower versions — so we could probably drop 4.0 and 5.0, but this version range is how it's configured upstream.